.github/workflows/push.yml

```diff
@@ -66,12 +66,6 @@ jobs:
           BRANCH_NAME=${BRANCH_NAME//./-}  # replace . with -
           BRANCH_NAME=${BRANCH_NAME//!/-}  # replace ! with -
           echo "GHA_DOCKER_PUSH_BRANCH_NAME=$BRANCH_NAME" >> $GITHUB_ENV
-      - id: verify-cloud-provider
-        shell: bash
-        if: env.GHA_DOCKER_PUSH_CLOUD_PROVIDER != 'GCP' && env.GHA_DOCKER_PUSH_CLOUD_PROVIDER != 'Azure'
```

Comment:
next time, reverse if values :-)

Reply:
live in the 80 do you? :yoda:

Reply:
seems like the parser did not like that for some reason, I'll leave it for another PR (have a go! ;) )